packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerForm/AddPrayerFormConnected.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
-import { Query } from 'react-apollo';
+import { Query, Mutation } from 'react-apollo';
 import { get } from 'lodash';
-import getUserProfile from 'newspringchurchapp/src/tabs/connect/getUserProfile';
+import GET_USER_PROFILE from 'newspringchurchapp/src/tabs/connect/getUserProfile';
```

Comment:
for the record, we aren't using this type of capitalization anywhere else in the app for queries or mutation.

Reply:
Yea I saw that, but the convention makes sense and I think we should adopt it, not only because it's what Apollo prescribes, but so you don't confuse it with the function argument on the Mutation component. Like this:

```
<Mutation mutation={getPrayers}>
    {(getPrayers) => (
        <Stuff />
    )}
</Mutation>
```

They are two completely different things but that's a little confusing as what's happening.

@conrad-vanl thoughts on why we avoided this convention all over the app?

Reply:
🤷‍♂

Reply:
Lets change this to camel case for now. Just so it stays consistent.